zerver/tests/test_user_groups.py

```diff
@@ -2001,7 +2003,7 @@ def check_update_user_group(
         promote_new_full_members()
         check_update_user_group(
             "help",
-            "Troubleshooting team",
+            "Troubleshooting team\n",
```

Comment:
Would be good to add a test for checking we don't embed url previews similar to the test we have for streams.